SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -443,6 +443,9 @@ constexpr uint32 SPATIAL_SNAPSHOT_SCHEMA_HASH = 679237978;
 constexpr uint32 SPATIAL_SNAPSHOT_VERSION_INC = 5;
 constexpr uint64 SPATIAL_SNAPSHOT_VERSION = ((((uint64)SPATIAL_SNAPSHOT_SCHEMA_HASH) << 32) | SPATIAL_SNAPSHOT_VERSION_INC);
 
+// Flag used to force the loading of the SpatialGDK module when testing with multiple processes
+const FString SpatialTestFlag = FString("-SpatialTest");
```

Comment:
```suggestion
constexpr char* SpatialTestFlag = TEXT("-SpatialTest");
```

Reply:
Is this what we usually do? I'd prefer the FString myself

Reply:
constexpr is compile time, no need initialise it, micro optimisation, pretty minor.

Reply:
Okay I'm not sure if we can use a constexpr here, at least not with the TEXT macro, as the compiler definitely complains about it. Using only "-SpatialTest" as the initialiser might solve this but I am not sure that will play nicely with how we are currently using it at the moment as in the following line will also make the compiler angry 

`AdditionalLaunchParamaters += " " + SpatialTestFlag " " `

Reply:
I would leave it as is, its not important.